webui/src/lib/components/auth/forms.jsx

```diff
@@ -9,9 +9,10 @@ import {SearchIcon} from "@primer/octicons-react";
 import {useAPI} from "../../hooks/api";
 import {Checkbox, DataTable, DebouncedFormControl, AlertError, Loading} from "../controls";
 
-const resolveEntityDisplayName = (ent) => {
+export const ResolveEntityDisplayName = (ent) => {
```

Comment:
In our JS code (and in React in general), capitalize React components, while functions should be camelCased - 
No matter if they are private or public.
(So this one should remain `resolveEntityDisplayName`.)

Reply:
More important:
Now that this func is exported, it seemed to me that it should reside in some `utils` file.
Well, apparently, there's already `utils.ts`, containing a `resolveDisplayName` func 😄 
Please unify these.

Reply:
@itaigilo, I talked with Idan F2F, and mentioned that this resolver method should be provided by the caller, given the fact that this form isn't aware of the caller and that it already mixes between groups and users

Reply:
Not sure this is in the context of this PR...
We can chat about it f2f if you'd like to.